packages/account/src/Components/currency-selector/radio-button-group.tsx

```diff
@@ -33,9 +29,8 @@ const RadioButtonGroup = ({
     is_fiat,
     item_count,
     description,
-    has_fiat,
 }: React.PropsWithChildren<TRadioButtonGroup>) => {
-    const [is_currency_selected, setIsCurrencySelected] = useState(false);
+    const [, setIsCurrencySelected] = useState(false);
```

Comment:
Why do we need the update state function?

Reply:
No need, I've removed the state totally.